src/app/(privateProfile)/analytics/analytics.module.scss

```diff
@@ -507,6 +507,108 @@
     }
 }
 
+.analyticsListOfOperationsWrapper {
+    margin-top: 20px;
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+}
+
+.analyticsListOfOperationsContainer {
+    display: flex;
+    align-items: end;
+    gap: 20px;
+    font-size: 2.0rem;
+    font-weight: 400;
+    line-height: 28px;
+    color: var(--color-dark-cyan-blue);
+}
+
+.analyticsListOfOperationsDownloadWrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+
+    &__title {
+        font-weight: 400;
+        font-size: 2.0rem;
+        line-height: 28px;
+        letter-spacing: 0px;
+        color: var(--color-very-dark-grayish-blue);
+    }
+}
+
+.analyticsListOfOperationsButtonContent {
+    display: flex;
+    gap: 20px;
+
+    &__PdfButton, &__XslButton {
+        width: 89px;
+        height: 52px;
+        background-color: var(--color-dark-spring-green);
+        color: var(--color-pure-light);
+        font-family: Rubik;
```

Comment:
font-family: "Rubik", Verdana, Arial, Tahoma, sans-serif;

can you change it here and in other similar places?